2.train-VAE/scripts/nbconverted/beta_vae_training.py

```diff
@@ -29,56 +28,117 @@
 # In[2]:
 
 
-# load the data 
-data_directory = "../0.data-download/data/"
-dfs = load_data(data_directory, adult_or_pediatric = "all")
-dependency_df = dfs[1]
-sample_df = dfs[0]
+# load the data
```

Comment:
In any notebooks or scripts that have stochastic elements, it is best to set a random seed at the top. https://www.geeksforgeeks.org/random-seed-in-python/

Please add a random seed to this notebook.